server/routes/index.ts

```diff
@@ -46,6 +46,7 @@ export default function routes(router: Router, services: Services): Router {
   get('/referrals/:id/rar-days', (req, res) => referralsController.viewRarDays(req, res))
   post('/referrals/:id/rar-days', (req, res) => referralsController.updateRarDays(req, res))
   get('/referrals/:id/check-answers', (req, res) => referralsController.checkAnswers(req, res))
+  post('/referrals/:id/check-answers', (req, res) => referralsController.sendDraftReferral(req, res))
```

Comment:
I feel this would be clearer as `post('/referrals/:id/send')`, as we're not updating a `check-answers` field, unlike the other `post`s here. You'd have to add an `action` to the form element in the view for this. Unless you feel that's more of a backend endpoint concern?

Reply:
Yeah, I think that makes sense, gonna change it.